ui/autocomplete.js

```diff
@@ -365,7 +365,7 @@ $.widget( "ui.autocomplete", {
 		}
 
 		if ( !element || !element[ 0 ] ) {
-			element = this.element.closest( ".ui-front" );
+			element = this.element.closest( ".ui-front,dialog" );
```

Comment:
I did a search and found all the selectors that weren't using spaces. Here's a PR: https://github.com/jquery/jquery-ui/pull/1521